pkg/controller/hawtio/hawtio_controller.go

```diff
@@ -298,10 +305,14 @@ func (r *ReconcileHawtio) Reconcile(request reconcile.Request) (reconcile.Result
 
 	if isOpenShift4 {
 		// Check whether client certificate secret exists
+
+		cronJob := &v1beta1.CronJob{}
+		cronJobName := "certificate-expiry-check"
```

Comment:
I think the CronJob name should be that of the `Hawtio` resource being reconciled, as you can have multiple `Hawtio` resources.